src/main/java/com/Repository/PlaceRepository.java

```diff
@@ -2,9 +2,11 @@
 
 import com.Model.Place;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 import org.springframework.stereotype.Component;
 import org.springframework.stereotype.Repository;
 
-public interface PlaceRepository extends CrudRepository<Place, Integer> {
+@RepositoryRestResource(collectionResourceRel = "place", path = "place")
```

Comment:
why do you need this?

Reply:
This is used to make crudRepository an endpoint. 
https://spring.io/guides/gs/accessing-data-rest/

Reply:
But it's not necessary if you're ok with the default path it creates. From the link you sent: 

>  @RepositoryRestResource is not required for a repository to be exported. It is used only to change the export details, such as using /people instead of the default value of /persons.

I'm in favor of relying on their defaults as much as possible.